java/src/main/java/com/amido/stacks/workloads/menu/api/v1/UpdateMenuController.java

```diff
@@ -1,72 +1,45 @@
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.*;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestAttribute;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
 
-@RequestMapping("/v1/menu/{id}")
+@RequestMapping(
+    path = "/v1/menu/{id}",
+    consumes = MediaType.APPLICATION_JSON_VALUE,
+    produces = MediaType.APPLICATION_JSON_VALUE + "; charset=utf-8",
+    method = RequestMethod.PUT)
 public interface UpdateMenuController {
 
   @PutMapping(consumes = "application/json", produces = "application/json; charset=utf-8")
   @Operation(
       tags = "Menu",
       summary = "Update a menu",
-      security = @SecurityRequirement(name = "bearerAuth"),
-      description = "Update a menu with new information",
-      responses = {
-        @ApiResponse(
-            responseCode = "200",
-            description = "Success",
-            content =
-                @Content(
-                    mediaType = "application/json",
-                    schema = @Schema(implementation = ResourceUpdatedResponse.class))),
-        @ApiResponse(
-            responseCode = "204",
-            description = "No Content",
-            content =
-                @Content(
-                    mediaType = "application/json",
-                    schema = @Schema(implementation = ErrorResponse.class))),
-        @ApiResponse(
-            responseCode = "400",
-            description = "Bad Request",
-            content =
-                @Content(
-                    mediaType = "application/json",
-                    schema = @Schema(implementation = ErrorResponse.class))),
-        @ApiResponse(
-            responseCode = "401",
-            description = "Unauthorized, Access token is missing or invalid",
-            content =
-                @Content(
-                    mediaType = "application/json",
-                    schema = @Schema(implementation = ErrorResponse.class))),
-        @ApiResponse(
-            responseCode = "403",
-            description = "Forbidden, the user does not have permission to execute this operation",
-            content =
-                @Content(
-                    mediaType = "application/json",
-                    schema = @Schema(implementation = ErrorResponse.class))),
-        @ApiResponse(
-            responseCode = "409",
-            description = "Conflict, an item already exists",
-            content =
-                @Content(
-                    mediaType = "application/json",
-                    schema = @Schema(implementation = ErrorResponse.class)))
-      })
+      description = "Update a menu with new information")
+  @ApiResponse(
```

Comment:
Same again, why not base?

Reply:
UpdateItem does not seem to have 409 status on the main branch (again not sure why). I believe we can add 409 and move this to base annotation?